sled-agent/src/zone_bundle.rs

```diff
@@ -2168,26 +2169,22 @@ mod illumos_tests {
     use super::StorageLimit;
     use super::Utf8Path;
     use super::Utf8PathBuf;
-    use super::Uuid;
     use super::ZoneBundleCause;
     use super::ZoneBundleId;
     use super::ZoneBundleInfo;
     use super::ZoneBundleMetadata;
     use super::ZoneBundler;
-    use super::ZFS;
     use anyhow::Context;
     use chrono::DateTime;
     use chrono::TimeZone;
     use chrono::Timelike;
     use chrono::Utc;
-    use illumos_utils::zpool::ZpoolName;
     use rand::RngCore;
-    use sled_storage::disk::RawDisk;
-    use sled_storage::disk::SyntheticDisk;
-    use sled_storage::manager::{FakeStorageManager, StorageHandle};
```

Comment:
FYI @bnaecker I did some work here to avoid depending on the `FakeStorageManager` -- instead, I'm using the new `StorageManagerTestHarness`, but trying to preserve the functionality of these tests.